src/components/Modals/ContactInfoTitle.jsx

```diff
@@ -14,6 +13,9 @@ import { fullContactPropTypes } from '../ContactPropTypes'
 import ContactIdentity from '../ContactCard/ContactIdentity'
 import GroupsSelect from '../GroupsSelect/GroupsSelect'
 import Control from '../GroupsSelect/SelectBox/Control'
+import Button from 'cozy-ui/transpiled/react/Buttons'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import { Grid } from '@material-ui/core'
```

Comment:
You should never import from `material-ui` directly. Can't you use https://docs.cozy.io/cozy-ui/react/#/Content?id=grid ?

Reply:
Also, why do we need to have a `Grid` now? Is this really because of `buttons`?

Reply:
I do not have any preference